### PR TITLE
SAN: Discharges

### PR DESCRIPTION
**Pull request by cameron-eyds**
*Issue #:* /bcgov/entity#15190

*Description of changes:*
- Include read-only Securities Act Notices in Discharge Flow
- Pre-populate Discharge Registering party for SE

Screenshot 2024-06-06 at 11 53 58 AM

Screenshot 2024-06-06 at 11 54 40 AM

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of the PPR license (Apache 2.0).

### CODE REVIEW

**Comment by bcregistry-sre** [comment]
Temporary Url for review: https://bcregistry-assets-dev--pr-1919-1bi1shyc.web.app

**Review by doug-lovett — APPROVED** [review]
Looks good!

File: ppr-ui/src/views/discharge/DischargeRegistration.vue [hunk 1]
```diff
@@ -54,26 +85,26 @@
           <h3 class="pt-6">
             Original Registering Party
           </h3>
-          <registering-party-summary
+          <RegisteringPartySummary
```
Comment (dimak1) [inline ppr-ui/src/views/discharge/DischargeRegistration.vue:88]: 👍🏻

**Review by dimak1 — APPROVED** [review]
LGTM!

**Pull request merged**